integrations/clickup/clickup/client.py

```diff
@@ -0,0 +1,64 @@
+from enum import StrEnum
+import typing
+from typing import Any, AsyncGenerator
+
+from httpx import Timeout
+from loguru import logger
+
+from port_ocean.context.event import event
+from port_ocean.context.ocean import ocean
+from port_ocean.utils import http_async_client
+
+
+class ObjectKind(StrEnum):
+    TEAM = "team"
+    PROJECT = "project"
+    ISSUE = "issue"
+
+class ClickupClient:
+    def __init__(self, clickup_api_token: str) -> None:
+        self.clickup_api_url = "https://api.clickup.com/api/v2"
+        self.clickup_api_token = clickup_api_token
+
+        self.api_auth_header = {"Authorization": self.clickup_api_token}
+        self.client = http_async_client
+        self.client.headers.update(self.api_auth_header)
+
+        self.client.timeout = Timeout(30)
+
+    # gets the list of teams, kind=team
```

Comment:
No need for the comments when the method definition is clear enough

Reply:
Comments removed